agent/bench-scripts/tests/pbench-trafficgen/test-51.txt

```diff
@@ -0,0 +1,310 @@
++++ Running test-51 pbench-trafficgen
+tool trigger strings for start: "Starting binary-search" and for stop: "Finished binary-search" are now registered for tool group: "default"
+found IOMMU option, continuing
+found dpdk-tools, continuing
+Total number of benchmark iterations: 4
+cp: cannot stat '/var/tmp/pbench-test-bench/pbench-agent/tmp/trex_cfg.yaml': No such file or directory
+cp: cannot stat 'profile1': No such file or directory
```

Comment:
Was it supposed to fail?

Reply:
No, I have added mock profile files to the test.